database/database.sql

```diff
@@ -0,0 +1,44 @@
+DROP DATABASE IF EXISTS jamusic;
+CREATE DATABASE jamusic;
+USE jamusic;
+DROP TABLE IF EXISTS profile;
```

Comment:
if you drop the database, no need to drop the tables: pick one ;)

Reply:
Fixed it, removed the drop table instructions.